src/handlers/uniswap.ts

```diff
@@ -51,11 +51,25 @@ export const getTestnetUniswapPairs = (
     network,
     {}
   );
-  const loweredPairs = mapKeys(pairs, (_, key) => toLower(key));
-  return mapValues(loweredPairs, value => ({
-    ...value,
-    address: toLower(value.address),
-  }));
+
+  const loweredPairs = Object.entries(pairs).reduce<Record<string, Asset>>(
```

Comment:
How about a name `keysToLowerCase`?

Reply:
usually, I don't change the old naming, but yea, in this case, it makes sense to update it